gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-environments.yaml

```diff
@@ -599,6 +599,12 @@ components:
                 payload:
                     type: string
                     description: The ruleset payload
+                format:
+                    type: string
+                    enum:
+                        - GRAVITEE_FEDERATION
+                        - GRAVITEE_MESSAGE
+                        - GRAVITEE_PROXY
```

Comment:
I think we need a description to explain that isn't need for standard specification